### PR TITLE
Feature/adrodo2 0 55

### PR DESCRIPTION
**Pull request by skoklowski**
- Prevent layout from collapsing vendor links in case of "display: inline-block" style
- Polish translation

### CODE REVIEW

File: src/components/popup/details/purposes/purposes.jsx [hunk 1]
```diff
@@ -161,10 +161,10 @@ export default class Purposes extends Component {
 						<div class={style.body}>
 							<LocalLabel localizeKey={`${currentPurposeLocalizePrefix}.description`} />
 							{!purposeIsTechnical &&
-							<a class={style.vendorLink} onClick={this.createOnShowVendors({ isCustom: false })}><LocalLabel localizeKey='showVendors'>Show full vendor list</LocalLabel></a>
+							<div><a class={style.vendorLink} onClick={this.createOnShowVendors({ isCustom: true })}><LocalLabel localizeKey='showCustomVendors'>Show full custom vendor list</LocalLabel></a></div>
 							}
 							{!purposeIsTechnical &&
```
Comment (ODudek) [inline src/components/popup/details/purposes/purposes.jsx]: Są dwa razy te same ify, można to po prostu ograć fragmentem czyli
```
{!purposeIsTechnical &&
<>
<div><a class={style.vendorLink} onClick={this.createOnShowVendors({ isCustom: true })}><LocalLabel localizeKey='showCustomVendors'>Show full custom vendor list</LocalLabel></a></div>
<div><a class={style.vendorLink} onClick={this.createOnShowVendors({ isCustom: false })}><LocalLabel localizeKey='showVendors'>Show full vendor list</LocalLabel></a></div>
</>
}
```

Comment (skoklowski) [reply src/components/popup/details/purposes/purposes.jsx]: Unfortunately it doesn't work because we use Preact not React.

Comment (skoklowski) [reply src/components/popup/details/purposes/purposes.jsx]: I've just implemented it in different way
```jsx
{!purposeIsTechnical && [
<div>...</div>,
<div>...</div>
]}
```

Comment (ODudek) [reply src/components/popup/details/purposes/purposes.jsx]: Based on their [documentation](https://preactjs.com/guide/v10/components/#fragments) fragments are available in the same way as in react. But the array is good too :)

**Review by ODudek — APPROVED** [review]

**Pull request merged**